packaging/packager

```diff
@@ -56,41 +56,35 @@ if ! type zip > /dev/null 2>&1; then
     exit 1
 fi
 
-function pluck_so_files() {
+function find_so_files() {
     sed -E '/\.so$|\.so\.[0-9]+$/!d'
 }
 
 function package_libc_alpine() {
-    if grep --fixed-strings "Alpine Linux" < /etc/os-release > /dev/null; then
+    if grep -F "Alpine Linux" < /etc/os-release > /dev/null; then
```

Comment:
I'm grudgingly OK with this. But please add a comment. I, for one, don't know what -F stands for and I have to look it up.

Reply:
In this particular case it seems like the long form name of the flag doesn't explain its function too well either.

I added a comment that explains it: c1480f7